token-generator/src/main/java/uk/gov/hmcts/auth/provider/service/token/HttpComponentsBasedServiceTokenGenerator.java

```diff
@@ -36,11 +39,19 @@ public String generate() {
         try {
             String oneTimePassword = totpAuthenticator.issueOneTimePassword(totpKey);
 
+            Map<String, String> tokenDetails = ImmutableMap.of(
+                "microservice", microservice,
+                "oneTimePassword", oneTimePassword
+            );
+
             HttpPost request = new HttpPost(baseUrl + "/lease");
-            request.setEntity(new UrlEncodedFormEntity(asList(
-                new BasicNameValuePair("microservice", microservice),
-                new BasicNameValuePair("oneTimePassword", oneTimePassword)
-            )));
+            request.setEntity(
+                new StringEntity(
+                    new ObjectMapper()
+                        .writeValueAsString(tokenDetails)
+                ));
+
+            request.addHeader("content-type", "application/json");
```

Comment:
`Content-Type`
Or even better use constants from a library. (I think apache commons is already included and has it)